app/assets/stylesheets/report.scss

```diff
@@ -0,0 +1,34 @@
+.report {
+  background: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.